RecoEgamma/EgammaTools/src/SCEnergyCorrectorSemiParm.cc

```diff
@@ -0,0 +1,316 @@
+//#include <TFile.h>
+#include "../interface/SCEnergyCorrectorSemiParm.h"
+#include "CondFormats/EgammaObjects/interface/GBRForestD.h"
+#include "CondFormats/DataRecord/interface/GBRDWrapperRcd.h"
+#include "FWCore/Framework/interface/ESHandle.h" 
+#include "DataFormats/EgammaCandidates/interface/Photon.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectron.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
+#include "RecoEcal/EgammaCoreTools/interface/EcalClusterTools.h"
+#include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "RooWorkspace.h"
```

Comment:
I think all Roo\* includes are not needed. Please check and clarify if it is not the case.